.claude/skills/ui-qa/references/harness.md

```
@@ -0,0 +1,83 @@
+
+## Starting
+
+```bash
+# From the repo root (requires Docker; takes 60–90s)
+uv run python scripts/hassette_demo.py
+```
+
+Run it in the background and poll its output for readiness. It prints machine-parseable
+lines when up:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)